planner/core/preprocess.go

```diff
@@ -327,6 +343,8 @@ const (
 	TypeRepair
 	// TypeShow for ShowStmt
 	TypeShow
+	// TypeExecute for ExecuteStmt
+	TypeExecute
```

Comment:
The new enum should be handled in `preprocessor.stmtType`.

Reply:
got, removed

Reply:
I was mean the function `preprocessor.stmtType` will transfer the enum to correspond string, if you add a new type here, the stringify function should also be updated.

Reply:
Soga, Got.
Before I tried to use this flag to mark a ast node as `Execute`, but latter I found it can be done with node.(ast.Execute). So it can be eliminated here. It's ok.